Dixit_Logic/Classes/Hand.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Dixit_Logic.Interfaces;
+
+namespace Dixit_Logic.Classes
+{
+    /// <summary>
+    /// This class represnts the card deck what a player hold in his/her hands.
+    /// </summary>
+    class Hand : IDeck
+    {
+        /// <summary>
+        /// It stores the list of cards what the hand contain.
+        /// </summary>
+        private List<ICard> _cards;
+
+        /// <summary>
+        /// Return with the cards what stores in _cards
+        /// </summary>
+        public IList<ICard> Cards
+        {
+            get
+            {
+                return _cards;
+            }
+        }
+
+        /// <summary>
+        /// Construct an empty hand
+        /// </summary>
+        public Hand()
+        {
+            _cards = new List<ICard>();
+        }
+
+        /// <summary>
+        /// Construct a hand from the given cards.
+        /// </summary>
+        /// <param name="cards">An IList implementation what contains card(ICard)s</param>
+        public Hand(IList<ICard> cards)
+        {
+            _cards = new List<ICard>();
+
+            foreach (ICard card in cards)
+            {
+                _cards.Add(card);
+            }
+        }
+
+        /// <summary>
+        /// That is the only way to put cards in the hand after the Hand 
+        /// object is constructed.
+        /// </summary>
+        /// <param name="cards">An IList implementation what contains card(ICard)s</param>
+        public void AddCards(IList<ICard> cards)
```

Comment:
I think there should be a method which adds a Single ICard. In each turn the player plays a Card, and then Draws a single one. With this method only you will have to wrap these single values into Lists.